content/blog/high-utilization-in-systems/index.md

```diff
@@ -0,0 +1,92 @@
+
+## High utilization
+
+We can define high utilization as team members spending _most of their time_ tackling tasks directly related to their projects. You can think of these tasks as something that _moves the project forward towards completion_. In most cases this will mean feature delivery, but it also applies to operational teams taking care of areas like infrastructure or security.
+
+Everyone has a queue of tasks, and this queue only grows as the number of tasks increases. I'm sure most of you have been in this position before, where it doesn't matter the amount of work you do, there's always new things in the queue. This seems great from a company or management perspective since it implies things are getting done. Or is it? I will argue that it's the opposite. The higher the utilization the longer teams will take to complete their tasks and deliver value.
+
+Again, I know this seems counterintuitive.
+
+## Cycle time
+
+We will use the common definition of cycle time here, which is:
+
+> The cycle time is the amount of time it takes to complete a specific task from start to finish.
+
+If you have more tasks than you can handle it's a given that something will have to wait in the queue, but why would cycle time increase?
+
+**Prioritization**
+
+Increased cognitive load is a big part of the problem.
+
+It's easy to prioritize among a small number of tasks, but it becomes increasingly harder to prioritize a higher number of them. Should you unblock a team that has been waiting for a few weeks, run a security review for an important product release or validate the approach that a team is planning to take on their technical design? Wait, what about this new incident that just started?
+
+It is time consuming to reason about what should be done next and it's likely that the team will have to prioritize for the short-term. This is not beneficial from a business perspective and tends to make things worse in the long run.
+
+**Context switching**
+
+It's no surprise that people are more productive when they can work on a task from start to finish without interruption, but as the utilization goes up, so does the amount of concurrent tasks people are tackling. Unfortunately switching context is _expensive_, energy and time-wise. This will not only decrease the throughput of the team, but also decrease the overall quality of the final product.
+
+**Communication**
+
+One of the first things that breaks down is communication. Asynchronous messages start taking longer to be answered or not answered at all in public and private channels. Emails or issues will tend to be forgotten until someone volunteers or gets "voluntold". You will notice less engagement in meetings, with people multitasking or not participating at all. This in turn decreases the amount of collaboration and trust among team members.
+
+Communication also includes critical areas such as:
+
+* Mentorship
+* Knowledge sharing
+* Code reviews
+* Documentation
```

Comment:
I disagree that mentorship and code reviews _really_ fall under communication. I know you were looking for a good name. How about "Company's long-term interests" or like "Long-term impactful tasks". 😁

Reply:
That's fair. I need to rearrange this section under things that are caused by high utilization versus things that are made worse by it. 😄